tutorials/high_performance_networking/README.md

```diff
@@ -0,0 +1,2171 @@
+## High Performance Networking with Holoscan
+
+This tutorial demonstrates how to use the advanced networking Holoscan operator (often referred to as ANO or `advanced_network` in HoloHub) for low latency and high throughput communication through NVIDIA SmartNICs. With a
```

Comment:
Should we have a leading sentence describing why the ANO exists? The intro mentions what it's for, but it doesn't mention why not just use one of the other libraries directly. It's kind of mentioned below in the managers section, but I wasn't sure if we want to lead with that.

Reply:
Sounds good